test/unit/helpers/xhr/MFA_ENROLL_allFactors.js

```diff
@@ -253,7 +253,42 @@ define({
        "profile":{
           "user":"inca@clouditude.net"
        }
-    }]
+    }, {
+        "enrollment": "OPTIONAL",
+        "status": "NOT_SETUP",
+        "factorType": "token:hotp",
+        "provider": "CUSTOM",
+        "profiles": [{
+          "id": '123',
+          "name": 'Entrust',
+          "_embedded": {
+            "enrolledFactors": []
+          }
+        }, {
+          "id": '124',
+          "name": 'Entrust2',
+          "_embedded": {
+            "enrolledFactors": []
+          }
+        }],
+        "policy": {
```

Comment:
won't this be in the profile?

Reply:
+1 to that comment. I thought profiles will have the cardinality data... or is that not yet true?

Reply:
yeah it would be in profile. Just removing it for now. No cardinality support yet for hotp anyway